### PR TITLE
add missing itunes tags

### PR DESCRIPTION
**Pull request by tadopionero**

### CODE REVIEW

**Comment by vanhuyz** [comment]
Note: this PR is a clone from https://github.com/lkiesow/python-feedgen/pull/97

File: Makefile [hunk 1]
```diff
@@ -47,8 +47,3 @@ doc-latexpdf:
 
 publish:
 	twine upload dist/*
-
-test:
-	coverage run --source=feedgen -m unittest discover -s tests
-	flake8 $$(find setup.py tests feedgen -name '*.py')
-	bandit -r feedgen
```
Comment (shoito) [inline Makefile]: @tadopionero cc @vanhuyz Why are you deleting this?

Comment (shoito) [reply Makefile]: I want to follow the coding rules and fix the following problems.
```
flake8 $(find setup.py tests feedgen -name '*.py')
feedgen/ext/podcast_entry.py:100:1: W293 blank line contains whitespace
feedgen/feed.py:133:13: E741 ambiguous variable name 'l'
feedgen/entry.py:118:27: E741 ambiguous variable name 'l'
feedgen/entry.py:139:13: E741 ambiguous variable name 'l'
feedgen/entry.py:452:17: E741 ambiguous variable name 'l'
make: *** [test] Error 1
```

`l` -> `al`

File: .travis.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ install:
   - pip install dist/feedgen*
 
 script:
-  - make test
+  # - make test
```
Comment (shoito) [inline .travis.yml]: ref https://github.com/pitpa/python-feedgen/pull/1/files#r813595205

File: Makefile (hunk 1 above)
Comment (tadopionero) [reply Makefile]: I readded it into makefile (I removed it before just for testing)

File: .travis.yml (hunk 2 above)
Comment (tadopionero) [reply .travis.yml]: I readded it into this file (I removed it before just for testing)

File: feedgen/ext/podcast_entry.py [hunk 3]
```diff
@@ -77,6 +81,23 @@ def extend_rss(self, entry):
...
+            episode.text = str(self.__itunes_episode)
+
+        if self.__itunes_title:
+            title = xml_elem('{%s}title' % ITUNES_NS, entry)
+            title.text = self.__itunes_title
+
+        if self.__itunes_episode_type in ('full', 'trailer', 'bonus'):
+            episode_type = xml_elem('{%s}episodeType' % ITUNES_NS, entry)
+            episode_type.text = self.__itunes_episode_type
+
```
Comment (shoito) [inline feedgen/ext/podcast_entry.py]: fix `feedgen/ext/podcast_entry.py:100:1: W293 blank line contains whitespace`
```suggestion

```

**Review by shoito — APPROVED** [review]

**Pull request merged**